src/AutoRest.CSharp.V3/CodeGen/ClientWriter.cs

```diff
@@ -47,9 +47,9 @@ public bool WriteClient(CodeWriter writer, ServiceClient operationGroup)
         private void WriteOperation(CodeWriter writer, ClientMethod operation, CSharpNamespace @namespace)
         {
             //TODO: Handle multiple responses
-            var schemaResponse = operation.ResponseType;
-            CSharpType? responseType = schemaResponse != null ? _typeFactory.CreateType(schemaResponse) : null;
-            CSharpType returnType = schemaResponse != null && responseType != null
+            ResponseBody? responseBody = operation.ResponseBodies.FirstOrDefault();
```

Comment:
If we don't use the array in generator let's not add an array to the model yet.

Reply:
Fixed.